guides/common/modules/con_restricting-hosts-access-to-content.adoc

```diff
@@ -14,7 +14,17 @@ For more information about lifecycle environments, see xref:Managing_Application
 
 Content overrides::
 By default, content hosted by {Project} can be either enabled or disabled.
+ifdef::orcharhino[]
+ifdef::red_hat_enterprise_linux[]
```

Comment:
I am guessing this is an attribute that you define downstream. If it is possible to rename that attribute, you can simply rename it to orcharhino_red_hat_enterprise_linux, and then it is sufficient to just check for that attribute instead of checking for orcharhino as well.

Reply:
yeah, unfortunately I cannot rename this easily. That's why I went with the double "ifdef" directives. I did not experimenting with "and" and "or" to combine ifdefs, but this would not work for all cases in a clean way in my local tests.